services/contentService.js

```diff
@@ -1,16 +1,30 @@
 const axios = require("axios");
+const fs = require("fs");
 
 exports.getHomePages = async function () {
-  const url = process.env.CMS_API + "/home-pages";
-  return await axios.get(url)
-    .then(function (response) {
-      // handle success
-      console.log(response.data);
-      return response.data;
-    }).catch(function (error) {
-    // handle error
-    console.log(error);
-    throw error;
-  })
-
+    console.log(process.env.MOCK_API_RESPONSES)
+    if (process.env.MOCK_API_RESPONSES !== true) {
```

Comment:
we should update the readme aswell, mentioning this variable, and the json file and what they do